src/runner/Resources.resx

```diff
@@ -116,8 +116,11 @@
     <value>Exit</value>
     <comment>Exit as a verb, as in Exit the application</comment>
   </data>
+  <data name="HIDE_TRAY_ICON_MENU_TEXT" xml:space="preserve">
+	<value>Hide Icon</value>
```

Comment:
Why the capital "I"?

Reply:
I was going for title case options in the context menu since most of the programs I have used do this. If we want to stick to title case that isn't a problem.

Reply:
It should be sentence case :)